### DISCUSSION TITLE
Why is my self-hosted umami not tracking Regions data?

### DISCUSSION THREAD

**Discussion by dd-oss** [umami-software/umami#2397] [Q&A]
Why is my self-hosted umami not tracking Regions data?

**Comment by SmartPhoneLover** [comment]
In your Cloudflare account: _Dashboard > DOMAIN> Rules > Transform Rules > Managed Transforms (tab)_, and enable the **Add visitor location headers** option.

**Reply by dd-oss** [reply]
Resolved, thank you!
